packages/fxa-settings/src/components/TermsPrivacyAgreement/index.tsx

```diff
@@ -10,19 +10,21 @@ import LinkExternal from 'fxa-react/components/LinkExternal';
 export type TermsPrivacyAgreementProps = {
   isPocketClient?: boolean;
   isMonitorClient?: boolean;
-  isDesktopRelay?: boolean;
+  isRelayClient?: boolean; // Relay is oauth RP
+  isDesktopRelay?: boolean; // `service=relay` on Fx desktop client ID
 };
 
 const TermsPrivacyAgreement = ({
   isPocketClient = false,
   isMonitorClient = false,
+  isRelayClient = false,
   isDesktopRelay = false,
 }: TermsPrivacyAgreementProps) => {
   return (
     <div
       className={`text-grey-500 text-xs ${isDesktopRelay ? 'mt-8' : 'mt-5'}`}
     >
-      {isPocketClient || isMonitorClient || isDesktopRelay ? (
+      {isPocketClient || isMonitorClient || isDesktopRelay || isRelayClient ? (
```

Comment:
Since we check for so many of these now, I think we might want a follow up issue to just take in `clientId` here and let this component do the checks. The way we have it now does make storybook and tests easier for this component, though. (We'd just be moving some mocking from other components into mocking for this.)

Reply:
Since three of these conditions display terms for subscription services, we could have a `isSubscriptionsService` function

Reply:
Sure, though not all subscription services want this displayed and `isDesktopRelay` is kind of a weird one. Maybe just `shouldShowSubscriptionTOS` or something 🤷‍♀️